src/mars_patcher/item_patcher.py

```diff
@@ -4,12 +4,14 @@
 from mars_patcher.rom import Rom
 from mars_patcher.room_entry import RoomEntry
 from mars_patcher.tileset import Tileset
+from mars_patcher.constants.reserved_space import ReservedConstants
 
-# keep these in sync with base patch
-MINOR_LOCS_ADDR = 0x7FF000
-MAJOR_LOCS_ADDR = 0x7FF200
-TANK_INC_ADDR = 0x7FF220
-METROID_COUNT_ADDR = 0x7FF227
+MINOR_LOCS_TABLE_ADDR = ReservedConstants.MINOR_LOCS_TABLE_ADDR
+MINOR_LOCS_ARRAY_ADDR = ReservedConstants.MINOR_LOCS_ARRAY_ADDR
+MAJOR_LOCS_ADDR = ReservedConstants.MAJOR_LOCS_ADDR
+MINOR_LOC_SIZE = 0x8    # Needs to be kept in sync with base patch
```

Comment:
Because it's not located in the reserved constant space. it's not even located anywhere in the rom (and thus intended to be changed); it's just the size of a specific struct: https://github.com/MetroidAdvRandomizerSystem/MARS-Fusion/blob/5f3fdff9d2b2b466f5df101c90e8a9c682197606/inc/structs.inc#L603

Although since we'll probably have a few of these, we should probably not make the reservedconstants class only contain stuff thats in the reserved space, but probably all kinds of constants that need to be kept up in sync with the base patch?